components/Typography/examples/TypographyMaterialStylesViewController.m

```diff
@@ -143,7 +169,13 @@ - (UITableViewCell *)tableView:(UITableView *)tableView
                                   reuseIdentifier:@"cell"];
   }
   cell.textLabel.text = _strings[indexPath.section];
-  cell.textLabel.font = _styleFonts[indexPath.row];
+  UIFont *font;
+  if (indexPath.row % 2) {
```

Comment:
if we put the font all in the same array we dont have to do any math here. See above.